src/components/ReactTodo/ReactTodo.js

```diff
@@ -60,12 +77,17 @@ function ReactTodo() {
       // Console.log the error's message
       console.log(error.message);
     }
+  }, [sort, url]);
+
+  // sort todo with toggle button
+    const handleSort = () => {
```

Comment:
Yes, yes, yes! I love the fact that you made this `sortOrder` state update the `todoList` as a side effect! This is some advanced level thinking that I just needed to call out! Great job! 👍 

On a side note, you can use the state setter callback function to toggle the `sortOrder` state:
```js
const handleSort = () => setSortOrder(prevSortOrder => prevSortOrder === 'asc' ? 'desc' : 'asc');
```